custom/icds_reports/management/commands/backfill_supervisor_id.py

```diff
@@ -0,0 +1,68 @@
+ucrs_with_awc_id = [
+    'config_report_icds-cas_static-pregnant-tasks_cases_6c2a698f',
+    'config_report_icds-cas_static-usage_forms_92fbe2aa',
+    'config_report_icds-cas_static-awc_mgt_forms_ad1b11f0'
+]
+
+child_health_ucrs = [
+    'config_report_icds-cas_dashboard_child_health_daily_fe_f83b12b7',
+    'config_report_icds-cas_static-dashboard_thr_forms_b8bca6ea',
+    'config_report_icds-cas_static-dashboard_growth_monitor_8f61534c',
+    'config_report_icds-cas_static-postnatal_care_forms_0c30d94e',
+    'config_report_icds-cas_static-complementary_feeding_fo_4676987e'
+]
+
+ccs_record_ucrs = [
+    'config_report_icds-cas_static-dashboard_birth_prepared_fd07c11f',
+]
+
+owner_id_ucrs = [
+    'config_report_icds-cas_static-hardware_cases_af8ee171', # owner_id
```

Comment:
I couldn't find a table to join with this. What UCR table has locations and owner_id columns?

Reply:
I don't think we added any columns to this one

Reply:
Oh yes, removing this one.